Crash on startup

```
07-24 15:29:58.137    9209-9257/edu.usf.cutr.opentripplanner.android V/OTP﹕ Attempt retrieving servers from sqlite
07-24 15:29:58.247    9209-9263/edu.usf.cutr.opentripplanner.android W/ActivityThread﹕ ClassLoader.loadClass: The class loader returned by Thread.getContextClassLoader() may fail for processes that host multiple applications. You should explicitly specify a context class loader. For example: Thread.setContextClassLoader(getClass().getClassLoader());
07-24 15:29:58.278    9209-9257/edu.usf.cutr.opentripplanner.android V/OTP﹕ [ 07-24 15:29:58.298   747: 1276 I/WindowManager ]
    Switching to real app window: Window{42aac148 u0 edu.usf.cutr.opentripplanner.android/edu.usf.cutr.opentripplanner.android.MyActivity}
07-24 15:29:58.368    9209-9257/edu.usf.cutr.opentripplanner.android V/OTP﹕ No data from sqlite. Attempt retrieving servers from google spreadsheet
07-24 15:29:58.438    9209-9209/edu.usf.cutr.opentripplanner.android W/IInputConnectionWrapper﹕ showStatusIcon on inactive InputConnection
07-24 15:29:58.478    9209-9209/edu.usf.cutr.opentripplanner.android I/Timeline﹕ Timeline: Activity_idle id: android.os.BinderProxy@4190bf00 time:5267839
07-24 15:29:59.779    9209-9257/edu.usf.cutr.opentripplanner.android W/dalvikvm﹕ threadid=23: thread exiting with uncaught exception (group=0x4166fce0)
07-24 15:29:59.779    9209-9257/edu.usf.cutr.opentripplanner.android E/AndroidRuntime﹕ FATAL EXCEPTION: AsyncTask #2
    Process: edu.usf.cutr.opentripplanner.android, PID: 9209
    java.lang.RuntimeException: An error occured while executing doInBackground()
            at android.os.AsyncTask$3.done(AsyncTask.java:300)
            at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:355)
            at java.util.concurrent.FutureTask.setException(FutureTask.java:222)
            at java.util.concurrent.FutureTask.run(FutureTask.java:242)
            at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231)
            at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
            at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
            at java.lang.Thread.run(Thread.java:841)
     Caused by: java.lang.ArrayIndexOutOfBoundsException: length=8; index=8
            at edu.usf.cutr.opentripplanner.android.tasks.ServerSelector.downloadServerList(ServerSelector.java:247)
            at edu.usf.cutr.opentripplanner.android.tasks.ServerSelector.doInBackground(ServerSelector.java:152)
            at edu.usf.cutr.opentripplanner.android.tasks.ServerSelector.doInBackground(ServerSelector.java:72)
            at android.os.AsyncTask$2.call(AsyncTask.java:288)
            at java.util.concurrent.FutureTask.run(FutureTask.java:237)
            at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231)
            at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
            at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
            at java.lang.Thread.run(Thread.java:841)
```

--- a/opentripplanner-android/src/main/java/edu/usf/cutr/opentripplanner/android/tasks/ServerSelector.java
+++ b/opentripplanner-android/src/main/java/edu/usf/cutr/opentripplanner/android/tasks/ServerSelector.java
@@ -16,8 +16,6 @@
 
 package edu.usf.cutr.opentripplanner.android.tasks;
 
-import com.google.android.gms.maps.model.LatLng;
-
 import android.app.Activity;
 import android.app.AlertDialog;
 import android.app.ProgressDialog;
@@ -32,6 +30,8 @@ import android.util.Log;
 import android.webkit.URLUtil;
 import android.widget.EditText;
 import android.widget.Toast;
+
+import com.google.android.gms.maps.model.LatLng;
 
 import java.io.BufferedReader;
 import java.io.IOException;
@@ -242,7 +242,7 @@ public class ServerSelector extends AsyncTask<LatLng, Integer, Integer>
                             break;
                         }
                     }
-                    if (allFieldsNotNull && (serverString.length >= 8)) {
+                    if (allFieldsNotNull && (serverString.length >= 9)) {
                         try {
                             Server s = new Server(currentTime, serverString[0], serverString[1],
                                     serverString[2], serverString[3], serverString[4],

